run_tests

```diff
@@ -0,0 +1,137 @@
+# Note that this script is taken from raven "raven/run_tests". Some changes were made to make
+#!/bin/bash
+echo 'Running FORCE tests ...'
+PYTHON_COMMAND=python
+SCRIPT_NAME=`readlink $0`
+if test -x "$SCRIPT_NAME";
+then
+    SCRIPT_DIRNAME=`dirname $SCRIPT_NAME`
+else
+    SCRIPT_DIRNAME=`dirname $0`
+fi
+SCRIPT_DIR=`(cd $SCRIPT_DIRNAME; pwd)`
+
+TEST_SET=0 # 0 for normal tests, 1 for plugin tests, 2 for both
```

Comment:
needed for FORCE tests?

Reply:
No, this is not needed. I cleaned up the run_tests file and the run_tests file is much smaller now.